File: HTTP_Server_CLI/scenarios/1.1.2-static_10.lua
-- @label: 10 static routes
-- @group: Static routes
-- @competitors: all
-- Round-robins through all 10 registered static routes.

local paths = {"/", "/about", "/contact", "/blog", "/pricing", "/docs", "/faq", "/terms", "/privacy", "/status"}
local counter = 0

request = function()
   counter = counter + 1
   return wrk.format("GET", paths[(counter % #paths) + 1])
end

File: HTTP_Server_CLI/scenarios/php/1.1.2-static_10.php
<?php
// @label: 10 static routes
// @group: Static routes
// @competitors: all
// Round-robins through all 10 registered static routes.

return [
   'method' => 'GET',
   'paths'  => ['/', '/about', '/contact', '/blog', '/pricing', '/docs', '/faq', '/terms', '/privacy', '/status'],
];
